tools/docker/boardfarm-ci/docker-compose.yml

```diff
@@ -58,3 +58,8 @@ services:
         user: ${CURRENT_UID}
         working_dir: $ROOT_DIR/tests
         entrypoint: ["bash", "run_bf_compose.sh"]
+
+networks:
+    default:
+        external:
+            name: dctest-network
```

Comment:
Just a quick note while I'm passing by: this needs to use the unique ID of the job as well (so CI_PIPELINE_ID does the trick in CI. Locally using the username is enough for most people).